javascript/packages/orchestrator/src/providers/podman/resources/tempoResource.ts

```diff
@@ -38,7 +38,7 @@ export class TempoResource {
         __dirname,
         `./configs/tempo.yaml`,
       );
-      await fs.writeFile(`${this.configPath}/tempo.yaml`, templateConfigPath);
+      await fs.copyFile(templateConfigPath, `${this.configPath}/tempo.yaml`);
```

Comment:
why is this change needed?

Reply:
Because writeFile doesn't works that way, it takes the content of the file, while copyFile take the path of the file